### PR TITLE
chore: Elm theme improvements on Learn Tab

### PR DESCRIPTION
**Pull request by saeedbashir**
### Description

Jira Ticket: [LEARNER-10163](https://2u-internal.atlassian.net/browse/LEARNER-10163)

Update the progress bar background color, caution button bg color and upgrade button bg color as suggested by the design team.

Config PR: https://github.com/edx/edx-mobile-config/pull/182

Light Theme:

| Before  | After |
| ------------- | ------------- |
| light_before | light_after|

Dark Theme

| Before  | After |
| ------------- | ------------- |
|  dark_before | dark_after |

### CODE REVIEW

File: Theme/Theme/Theme.swift [hunk 1]
```diff
@@ -73,6 +73,9 @@ public struct Theme {
         public private(set) static var courseCardShadow = ThemeAssets.courseCardShadow.swiftUIColor
         public private(set) static var courseCardBackground = ThemeAssets.courseCardBackground.swiftUIColor
         public private(set) static var deleteAccountBG = ThemeAssets.deleteAccountBG.swiftUIColor
+        public private(set) static var primarycardCautionBG = ThemeAssets.primaryCardCautionBG.swiftUIColor
+        public private(set) static var primarycardUpgradeBG = ThemeAssets.primaryCardCourseUpgradeBG.swiftUIColor
+        public private(set) static var primarycardProgressBG = ThemeAssets.primaryCardProgressBG.swiftUIColor
```
Comment (rnr) [inline Theme/Theme/Theme.swift]: `primaryCard...` (capitalised 'C') looks better for my view

**Review by rnr — APPROVED** [review]

File: Theme/Theme/Assets.xcassets/Colors/PrimaryCard/Contents.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```
Comment (shafqat-muneer) [inline Theme/Theme/Assets.xcassets/Colors/PrimaryCard/Contents.json:6]: Is this change intentional?

Comment (saeedbashir) [reply Theme/Theme/Assets.xcassets/Colors/PrimaryCard/Contents.json:6]: It auto-generates when we create a folder in the assets.

**Review by shafqat-muneer — APPROVED** [review]
